HelpSource/Classes/Ndef.schelp

```diff
@@ -4,7 +4,9 @@ categories:: Libraries>JITLib>NodeProxy, Live Coding
 related:: Classes/ProxySpace, Classes/Tdef
 
 description::
-Reference to a proxy, forms an alternative to link::Classes/ProxySpace::. All methods are inherited from link::Classes/NodeProxy::.
+Ndef builds a synth, the internals of which can be replaced while playing. An Ndef can be started, stopped and manipulated through its key. The old synth and its replacement can automatically crossfade and the replacement time can be quantized.
```

Comment:
I'm not sure about the "through its key" language here, really everything is manipulated through the Ndef instance. The key is just parameter to the Ndef constructor that indicates which object it is a proxy for. Also afaik it's not the Ndef that's being started, stopped etc, it's the synth for which it is a proxy